docs/examples/physics_using_pybullet/viz_domino.py

```diff
@@ -0,0 +1,228 @@
+# * Update the position and orientation.
+
+
+def sync_domino(object_index, multibody):
+    pos, orn = p.getBasePositionAndOrientation(multibody)
+
+    rot_mat = np.reshape(
+        p.getMatrixFromQuaternion(
+            p.getDifferenceQuaternion(orn, domino_orns[object_index])),
+        (3, 3))
+
+    vertices[object_index * sec: object_index * sec + sec] = \
+        (vertices[object_index * sec: object_index * sec + sec] -
+         domino_centers[object_index]) @ rot_mat + pos
+
+    domino_centers[object_index] = pos
+    domino_orns[object_index] = orn
+
+
+###############################################################################
+# Here, we define a textblock to display the Avg. FPS and simulation steps.
+
+fpss = np.array([])
+tb = ui.TextBlock2D(text="Avg. FPS: \nSim Steps: ", position=(0, 680),
+                    font_size=30, color=(1, 0.5, 0))
+scene.add(tb)
+
+###############################################################################
+# Set the camera for better visualization.
+
+scene.set_camera(position=(10.46, -8.13, 6.18),
+                 focal_point=(0.0, 0.0, 0.79),
+                 view_up=(-0.27, 0.26, 0.90))
+
+
+###############################################################################
+# Timer callback is created which is responsible for calling the sync and
+# simulation methods.
+
+
+# Create timer callback which will execute at each step of simulation.
+def timer_callback(_obj, _event):
+    global apply_force, fpss
+    cnt = next(counter)
+    showm.render()
+
+    if cnt % 1 == 0:
+        fps = scene.frame_rate
+        fpss = np.append(fpss, fps)
+        tb.message = "Avg. FPS: " + str(np.round(np.mean(fpss), 0)) + \
+                     "\nSim Steps: " + str(cnt)
+
+    # Get the position and orientation of the first domino.
+    domino1_pos, domino1_orn = p.getBasePositionAndOrientation(dominos[0])
+
+    # Apply force on the First Domino (domino) above the Center of Mass.
+    if apply_force:
+        # Apply the force.
+        p.applyExternalForce(dominos[0], -1,
+                             forceObj=[100, 0, 0],
+                             posObj=domino1_pos + np.array([0, 0, 1.7]),
+                             flags=p.WORLD_FRAME)
+        apply_force = False
+
+    # Updating the position and orientation of individual dominos.
+    for idx, domino in enumerate(dominos):
+        sync_domino(idx, domino)
+    utils.update_actor(domino_actor)
+
+    # Simulate a step.
+    p.stepSimulation()
+
+    # Exit after 2000 steps of simulation.
+    if cnt == 2000:
+        showm.exit()
+
+
+# Add the timer callback to showmanager.
+# Increasing the duration value will slow down the simulation.
+showm.add_timer_callback(True, 1, timer_callback)
+
+interactive = False
+
+# start simulation
+if interactive:
+    showm.start()
+
+window.record(scene, out_path="domnio_simulation.png", size=(900, 768))
+###############################################################################
```

Comment:
The `#####` line isn't necessary. also set `out_path="viz_domino.png"` to make it same as the other examples.